content/docs/1.0.0/iofogctl/iofogctl.md

```diff
@@ -0,0 +1,209 @@
+```
+
+Under the hood, `iofogctl` will also deploy ioFog Connector and Kubernetes services, which are another essential part of the ioFog stack
+
+We can now check that the Controller was deployed successfully.
+
+```console
+$ iofogctl get all
+
+CONTROLLER	STATUS		AGE		UPTIME
+my-ctrl		online		-		1s
+
+AGENT		STATUS		AGE		UPTIME
+
+MICROSERVICE	STATUS		AGE
+
+```
+
+It is also possible to query for all Kubernetes resources installed. The namespace `iofogctl` uses is always `iofog`, so we can run `kubectl get all -n iofog` to see all the resources installed on our cluster.
+
+## Connect Existing ioFog Stack With iofogctl
+
+Instead of deploying our own iofogctl stack, we can just connect to another Controller, for example one deployed by Helm. See [helm tutorial](../kubernetes/how-to-helm.html) for in depth dive into Helm.
+
+```bash
+iofogctl connect my-ctrl \
+  -o $(kubectl -n iofog get svc controller -o jsonpath='{.status.loadBalancer.ingress[0].ip}:{.spec.ports[0].port}') \
+  -e user@domain.com  -p '#Bugs4Fun'
+```
+
+It is also possible to use a kubeconfig file instead of specifying the Controller endpoint manually.
+
+```bash
+iofogctl connect my-ctrl --kube-config ~/.kube/conf --email EMAIL --pass PASSWORD
+```
+
+## Deploy Agent On The ioFog Stack
+
+We can now deploy Agents to a namespace that has a working Controller connected. Here, `my-first-agent` is the agent name, `1.2.3.4` is the external IP that iofogctl can access, and `/home/username/.ssh/agent-key` is the key that needs to be authorized on the agent.
+
+The node that will be hosting our Agent needs to be accessible via SSH, as it is the only way `iofogctl` can deploy the software and provision and register the Agent on that node.
+
+```bash
+iofogctl deploy agent my-first-agent --user username --host 1.2.3.4 --key-file /home/username/.ssh/agent-key
+```
+
+We can now check that the Agent was deployed successfully.
+
+```console
+$ iofogctl get all
+
+CONTROLLER	    STATUS		AGE		UPTIME
+ctrl		    online		-		1s
+
+AGENT		    STATUS		AGE		UPTIME
+my-first-agent	RUNNING		18070d23h	0s
+
+MICROSERVICE	STATUS		AGE
+```
+
+### Deploy Agent Using YAML File
+
+We can use a YAML file to specify Controllers and Agents to deploy in lieu of the separate commands detailed above.
+
+Run `iofogctl deploy --help` to view an example of a valid YAML file. When the YAML file is ready, run the deploy command and specify your yaml filename.
+
+```bash
+iofogctl deploy -f platform.yaml
+```
+
+## View ioFog Cluster Details
+
+Now that we are connected to a live ioFog cluster, we can go ahead and do some introspection.
+
+Try to display individual resources or all resources within a namespaces with the get command:
+
+```bash
+iofogctl get controllers
+iofogctl get agents
+iofogctl get all
+```
+
+To get more detailed information, we can use the describe command:
+
+```bash
+iofogctl describe controller my-ctrl
+iofogctl describe agent my-first-agent
+```
```

Comment:
## Delete ioFog Cluster

You can delete all resources in a namespace.
```bash
iofogctl delete all
```
If you delete a Controller, it will also deprovision any Agents in the corresponding namespace
```bash
iofogctl delete controller my-ctrl
```
You can deprovision agents one by one also.
```bash
iofogctl delete agent my-agent
```